source/store/AppContext.js

```diff
@@ -20,18 +58,33 @@ function AppProvider({ children }) {
   const provider = {
     mode,
     handleSetMode: (newMode) => handleSetMode(newMode),
-    isDevMode: mode === 'development',
+    isDevMode: mode === "development",
   };
 
   useEffect(() => {
     StorageService.saveData(APP_ENV_KEY, mode);
   }, [mode]);
 
+  if (provider.isDevMode) {
```

Comment:
It might not be such a good idea to use this code because it will run on every re-render. But then the question is how often this context does it.

Reply:
Changed completely